2021-05-28 04:20:57,463 INFO    MainThread:29295 [wandb_setup.py:_flush():69] setting env: {}
2021-05-28 04:20:57,463 INFO    MainThread:29295 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-28 04:20:57,464 INFO    MainThread:29295 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210528_042057-1uen5uob/logs/debug.log
2021-05-28 04:20:57,464 INFO    MainThread:29295 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210528_042057-1uen5uob/logs/debug-internal.log
2021-05-28 04:20:57,464 INFO    MainThread:29295 [wandb_init.py:init():369] calling init triggers
2021-05-28 04:20:57,464 INFO    MainThread:29295 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'img_size': 5, 'batch_size': 64, 'init_lr': 0.001614813570468519}
2021-05-28 04:20:57,464 INFO    MainThread:29295 [wandb_init.py:init():418] starting backend
2021-05-28 04:20:57,464 INFO    MainThread:29295 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-28 04:20:57,475 INFO    MainThread:29295 [backend.py:ensure_launched():132] starting backend process...
2021-05-28 04:20:57,479 INFO    MainThread:29295 [backend.py:ensure_launched():136] started backend process with pid: 29304
2021-05-28 04:20:57,481 INFO    MainThread:29295 [wandb_init.py:init():423] backend started and connected
2021-05-28 04:20:57,482 INFO    MainThread:29295 [wandb_init.py:init():465] updated telemetry
2021-05-28 04:20:57,482 INFO    MainThread:29295 [wandb_init.py:init():484] communicating current version
2021-05-28 04:20:58,233 INFO    MainThread:29295 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.31 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-28 04:20:58,233 INFO    MainThread:29295 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-28 04:20:58,447 INFO    MainThread:29295 [wandb_init.py:init():522] starting run threads in backend
2021-05-28 04:20:58,483 INFO    MainThread:29295 [wandb_run.py:_console_start():1565] atexit reg
2021-05-28 04:20:58,483 INFO    MainThread:29295 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-28 04:20:58,483 INFO    MainThread:29295 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-28 04:20:58,487 INFO    MainThread:29295 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-28 04:20:58,487 INFO    MainThread:29295 [wandb_init.py:init():546] run started, returning control to user process
2021-05-28 04:23:55,543 INFO    MainThread:29295 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 1
2021-05-28 04:23:55,544 INFO    MainThread:29295 [wandb_run.py:_restore():1507] restore
2021-05-28 04:23:57,757 INFO    MainThread:29295 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-28 04:23:58,525 INFO    MainThread:29295 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 789
}

2021-05-28 04:23:59,057 INFO    MainThread:29295 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4060
  total_bytes: 4060
}

2021-05-28 04:23:59,159 INFO    MainThread:29295 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4060
  total_bytes: 4060
}

2021-05-28 04:23:59,260 INFO    MainThread:29295 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4060
  total_bytes: 4060
}

2021-05-28 04:23:59,362 INFO    MainThread:29295 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4060
  total_bytes: 4060
}

2021-05-28 04:23:59,463 INFO    MainThread:29295 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4060
  total_bytes: 4060
}

2021-05-28 04:23:59,564 INFO    MainThread:29295 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4060
  total_bytes: 4060
}

2021-05-28 04:23:59,665 INFO    MainThread:29295 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4060
  total_bytes: 4060
}

2021-05-28 04:24:00,754 INFO    MainThread:29295 [wandb_run.py:_show_summary():1812] rendering summary
2021-05-28 04:24:00,754 INFO    MainThread:29295 [wandb_run.py:_show_history():1850] rendering history
2021-05-28 04:24:00,755 INFO    MainThread:29295 [wandb_run.py:_show_files():1879] logging synced files
